### PR TITLE
ci: Template for bug reporting, upgrade python version for codspeed

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
## What type of PR is this? (check all applicable)

- [ ] 💾 Refactor
- [ ] 🪄 Feature
- [ ] 🐞 Bug Fix
- [ ] 🔧 Optimization
- [ ] 📚 Documentation
- [ ] 🧪 Test
- [x] 🛠️ Other

## Related issues

- Related issue #
- Closes #

## Checklist

- [x] Code follows style guide
- [ ] Tests added
- [ ] Documented the changes

***

## Please explain your changes below.
Python version upgraded so codspeed "Trace generation" is possible: https://docs.codspeed.io/features/trace-generation/

### CODE REVIEW

File: .github/workflows/codespeed.yml
Comment (hameerabbasi) [inline .github/workflows/codespeed.yml:1]: Typo `codespeed`->`codspeed` in the filename.

**Review by hameerabbasi — CHANGES_REQUESTED** [review]
One minor change I missed in the last PR, otherwise is good to go.

File: .github/workflows/codespeed.yml
Comment (DeaMariaLeon) [reply .github/workflows/codespeed.yml:1]: Done.

**Review by hameerabbasi — APPROVED** [review]

**Comment by hameerabbasi** [comment]
This is in, thanks @DeaMariaLeon!

**Pull request merged**